src/com/itacademy/eshop/product/Product.java

```diff
@@ -2,19 +2,23 @@
 
 import com.itacademy.eshop.product.types.Category;
 
+import java.time.LocalDate;
 import java.util.ArrayList;
 
 public class Product {
     private String name;
     private double price;
     private Category category;
     private ArrayList<Review> reviews;
+    private Description description;
+    private Enum MISC;
```

Comment:
You do not need to have this generic Enum because you have updated the values of Category enum